src/frameworks/angular/index.js

```diff
@@ -0,0 +1,63 @@
+const inquirer = require('inquirer');
+const path = require('path');
+
+/* eslint-disable no-useless-escape */
+const packageJsonScripts = {
+  scripts: {
+    stylelint: 'stylelint \"**/*.scss\"',
```

Comment:
We need to add a dev script, per the README.md. Either reference a port from .env (so we can default it to 8080 like in the readme) or customize this port however `ng serve` requires. This might be as simple as `env-cmd .env npm start`